lib/hanami/view/rendering.rb

```diff
@@ -104,6 +105,25 @@ def render
           layout.render
         end
 
+        # It tries to invoke a method for the view or a local for the given key.
+        # If the lookup fails, it returns a null object.
+        #
+        # @return [Objeect,Hanami::View::Rendering::NullLocal] the returning value
+        #
+        # @since x.x.x
+        #
+        # @example
+        #   <% if local(:plan).overdue? %>
+        #     <h2>Your plan is overdue.</h2>
+        #   <% end %>
+        def local(key)
```

Comment:
@davydovanton `Rendering#respond_to?` isn't accurate.

If you remove this method, the other one takes precedence but the test fail. It isn't able to check if a local or a method exposed in a layout is available to the entire rendering process, hence the duplication. (sigh).